spec/javascripts/fixtures/json/middleware_topology_response.json

```diff
@@ -79,6 +95,14 @@
         "source": "MiddlewareManager1",
         "target": "MiddlewareServer1"
```

Comment:
is this link still relevant in the current architecture with eap/wildfly? I mean will we get it on real env? Won't we get only wildfly/eap objects in the json, no?

Reply:
Well I am not against removing these, however we still use MiddlewareServer for Hawkular server. So Whenever I spin hawkular standalone the server inside it will be of type MiddlewareServer. I tested this with hawkinit and it was still present.
However if you'd were to run hawkular provider on wildfly/eap solely by installing some packages and such (kinda hard to do) it will be of type wildfly/eap.

Reply:
ok, as long as it appears then it should be present. the point here is to make sure this reflects a real env.